0-bootstrap/main.tf

```diff
@@ -107,6 +116,16 @@ module "cloudbuild_bootstrap" {
   cloudbuild_plan_filename  = "cloudbuild-tf-plan.yaml"
   cloudbuild_apply_filename = "cloudbuild-tf-apply.yaml"
 
+  project_labels = {
+    environment       = "bootstrap"
+    application_name  = "cloudbuild-bootstrap"
+    billing_code      = "1234"
+    primary_contact   = "example1"
+    secondary_contact = "example2"
+    business_code     = "bu1"
```

Comment:
should we  use `abcd` since this project is not tied to an actual business unit ?